packages/core/src/constants.ts

```diff
@@ -1,5 +1,29 @@
-import { KuaiArguments } from './type'
+import { NetworkConfig } from './type'
+import { config } from '@ckb-lumos/lumos'
 
-export const DEFAULT_KUAI_ARGUMENTS: KuaiArguments = {
+export const DEFAULT_KUAI_ARGUMENTS = {
   network: 'docker-node',
 }
+
+export const DEFAULT_NETWORKDS: {
+  [name: string]: NetworkConfig
+} = {
+  mainnet: {
+    rpcUrl: 'https://mainnet.ckb.dev/rpc',
+    prefix: 'ckb',
+    scripts: config.predefined.LINA.SCRIPTS,
+  },
+  testnet: {
+    rpcUrl: 'https://testnet.ckb.dev/rpc',
+    prefix: 'ckt',
+    scripts: config.predefined.AGGRON4.SCRIPTS,
+  },
+  devnet: {
+    rpcUrl: 'http://localhost:8114',
```

Comment:
Better to use `127.0.0.1` because `localhost` will be resolved to `ipv6` since Node@17

Ref: https://github.com/nodejs/node/pull/39987